### PR TITLE
add cucumber test

### PR DESCRIPTION
**Pull request by aidatulahmad**

### CODE REVIEW

File: src/main/java/steps/AndroidLoginSteps.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public void tapOnLoginButton() throws Exception {
+        WebDriverManager.getAppiumDriver().findElement(By.id("textViewLogin")).click();
+        Logger.info("should see correct Login text button");
+    }
+
+    @And("user select and tap on Germany")
+    public void selectAndTapOnDropDown() throws Exception {
+        Logger.info("click on the drop-down button");
+        WebDriverManager.getAppiumDriver().findElement(By.xpath("//android.widget.TextView[contains(@text,'Germany')]")).click();
+        Thread.sleep(2000);
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:22]: Please do not use `Thread.sleep`s. We should already be able to bypass this with the Waits if necessary.

File: src/main/java/steps/AndroidLoginSteps.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    @And("user enter the username {string}")
+    public void userEnterUsername(String email) {
+        Logger.info("enter username");
+        WebDriverManager.getAppiumDriver().findElement(By.id("editTextEmail")).sendKeys(email);
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:28]: As you are already aware we do not use this king of syntax with our tests for reasons we are aware of already. If there's a need to identify elements using a specific driver such as `getAppiumDriver()` let's treat that in the framework

File: src/main/java/steps/AndroidLoginSteps.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public void tapOnSkipButton() throws InterruptedException {
+        Logger.info("tap skip button");
+        WebDriverManager.getAppiumDriver().findElement(By.id("skip")).click();
+        Thread.sleep(2000);
+    }
+
+    @Then("user should see the screen title as {string}")
+    public void shouldSeeTheScreenTitleAs(String title) throws Exception {
+        Logger.info("should see correct screen title");
+        Assertions.assertEquals(title, WebDriverManager.getAppiumDriver().findElements(By.className("android.widget.TextView")).get(0).getText());
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:54]: Same as above

**Review by leonardmustatea — CHANGES_REQUESTED** [review]

File: src/main/java/steps/AndroidLoginSteps.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import skynet.interactions.Wait;
+import skynet.utils.AHCustomAsserts;
+import skynet.utils.Logger;
+
+public class AndroidLoginSteps {
+    @When("user tap on the Login text button")
+    public void tapOnLoginButton() throws Exception {
+        Logger.info("tap Login text button");
+        WebElement el = Elements.findElement(By.id("textViewLogin"));
+        el.click();
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:18]: Please use the methods from the framework

File: src/main/java/steps/AndroidLoginSteps.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        Logger.info("click drop-down button");
+        WebElement el = Elements.findElement(By.xpath("//android.widget.TextView[contains(@text,'Germany')]"));
+        el.click();
+    }
+
+    @And("user enter the username {string}")
+    public void userEnterUsername(String email) throws Exception {
+        Logger.info("enter username");
+        WebElement el = Elements.findElement(By.id("editTextEmail"));
+        el.sendKeys(email);
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:32]: Please use the methods from the framework

File: src/main/java/steps/AndroidLoginSteps.java [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    @And("user enter the password {string}")
+    public void userEnterPassword(String password) throws Exception {
+        Logger.info("enter password");
```
Comment (leonardmustatea) [inline src/main/java/steps/AndroidLoginSteps.java:37]: Please update as per our discussion. Only method calls in step defs

File: src/test/java/features/authentication/AndroidTest.feature [hunk 7]
```diff
@@ -0,0 +1,21 @@
+@Epic=authenticationTest
+Feature: Feature - Android app Login
+
+#  @Regression
+  Scenario: 1562447 - Android app Login successful
+    When user tap on the Login text button
+    And user select and tap on Germany
+    And user enter the username "limangotesting@gmail.com"
```
Comment (leonardmustatea) [inline src/test/java/features/authentication/AndroidTest.feature:8]: Please extract these into variables using Scenario Outline and Exmaples table

**Review by leonardmustatea — CHANGES_REQUESTED** [review]

File: src/main/java/steps/AndroidLoginSteps.java (hunk 2 above)
Comment (aidatulahmad) [reply src/main/java/steps/AndroidLoginSteps.java]: step defs is updated

File: src/main/java/steps/AndroidLoginSteps.java (hunk 3 above)
Comment (aidatulahmad) [reply src/main/java/steps/AndroidLoginSteps.java]: updated

File: src/main/java/steps/AndroidLoginSteps.java (hunk 4 above)
Comment (aidatulahmad) [reply src/main/java/steps/AndroidLoginSteps.java]: updatede

File: src/main/java/steps/AndroidLoginSteps.java (hunk 5 above)
Comment (aidatulahmad) [reply src/main/java/steps/AndroidLoginSteps.java]: updated

File: src/main/java/steps/AndroidLoginSteps.java (hunk 6 above)
Comment (aidatulahmad) [reply src/main/java/steps/AndroidLoginSteps.java:31]: updated

File: src/test/java/features/authentication/AndroidTest.feature (hunk 7 above)
Comment (aidatulahmad) [reply src/test/java/features/authentication/AndroidTest.feature]: updated

**Review by leonardmustatea — APPROVED** [review]

**Review by leonardmustatea — APPROVED** [review]

**Review by leonardmustatea — APPROVED** [review]

**Review by leonardmustatea — APPROVED** [review]

**Pull request merged**